src/main/scala/pl/edu/agh/iet/akka_debugging/sbt/TracedActorsFinder.scala

```diff
@@ -0,0 +1,32 @@
+package pl.edu.agh.iet.akka_debugging.sbt
+
+import java.io.File
+
+import pl.edu.agh.iet.akka_debugging.sbt.ScalaClassDefinitionParser.parseClassDefinitions
+
+import scala.collection.mutable
+import scala.io.Source
+
+object TracedActorsFinder {
+  val TRAIT_NAME = "TracedActor"
```

Comment:
http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods

But for it doesn't matter right now.